chirho/observational/handlers/soft_conditioning.py

```diff
@@ -274,12 +225,8 @@ def configure(
             return None
 
         if msg["fn"].base_dist.v.is_floating_point():
-            scale = self.scale * functools.reduce(
-                operator.mul, msg["fn"].event_shape, 1.0
-            )
-            return KernelSoftConditionReparam(
-                RBFKernel(scale=scale, event_dim=len(msg["fn"].event_shape))
-            )
+            support = constraints.real
```

Comment:
You'll need to account for event dimensions here to make tests pass. I think you can just set `support = msg["fn"].base_dist.support`, which should be correct by construction.